packages/wallet-service/src/utils.ts

```diff
@@ -7,6 +7,7 @@
 
 import { BinaryToTextEncoding, createHash } from 'crypto';
 
+import { Logger } from 'winston';
```

Comment:
Not sure why this happened, but the build was failing because of this missing dependency

Reply:
I believe its used as an argument type, maybe the LSP does not count type annotations as usage of the import.